### PR TITLE
commodity language fix

### PR DESCRIPTION
**Pull request by joonicks**

### CODE REVIEW

**Comment by impaktor** [comment]
- Your editor is indenting empty lines, could you please configure it to not do that / remove trailing spaces?

- You need to add the strings to a en.json file for them to be seen by the translation system

**Comment by impaktor** [comment]
Once the changes are made, just do a git add

    git add data/ui-code/en.json
    git add data/ui/StationView/CommodityMarket.lua

Then _amend_ your previous commit (the one in this PR):

    git commit --amend

Changes are now squashed into the previous commit. If all looks well (`git show HEAD`, and `git log`) push to your branch (this PR), by force, since hash has changed:

    git push -f

**Comment by fluffyfreak** [comment]
So is this good to review now?

It's hard to follow the progress when people don't reply to say if they've updated a PR since we don't get any notification of changes themselves.

**Comment by impaktor** [comment]
@fluffyfreak yes, @joonicks pinged me about it, but I haven't gotten around to it yet. Please feel free to review / merge it if you like.

**Comment by joonicks** [comment]
ah yeah this should be the final commit for the translation thing
